src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -1097,8 +1083,7 @@ function initAugmentations() {
              "monitors and regulates nervous impulses coming to and from the spinal column, " +
              "essentially 'governing' the body. By doing so, it improves the functionality of the " +
              "body's nervous system.<br><br>" +
-             "This is a special augmentation because it can be leveled up infinitely. Each level of this augmentation " +
-             "increases ALL of the player's multipliers by 1%.",
+             "Each level increases all multipliers by 1%.",
```

Comment:
I would keep the part about it being possible for this Augmentation to be leveled up infinitely, as that is important information